packages/manager/apps/dedicated/client/app/account/kyc-documents/dashboard/template.html

```diff
@@ -145,6 +145,7 @@ <h4 data-translate="documents_dashboard_subtitle"></h4>
                     class="oui-button oui-button_primary oui-button_l text-light"
                     href="{{:: $ctrl.hubLink }}"
                     rel="noopener"
+                    data-ng-click="$ctrl.trackClick($ctrl.TRACKING_CTA_DASHBOARD)"
```

Comment:
You missed one update in the html template 

```suggestion
                    data-ng-click="$ctrl.trackClick($ctrl.TRACKING.CTA_DASHBOARD)"
```